django/core/handlers/asgi.py

```diff
@@ -263,7 +263,20 @@ async def read_body(self, receive):
                 raise RequestAborted()
             # Add a body chunk from the message, if provided.
             if "body" in message:
-                body_file.write(message["body"])
+                # Perform IO in another thread if the temp file has rolled over
+                # to disk.
+                on_disk = getattr(body_file, "_rolled", False)
+                if on_disk:
+                    async_write = sync_to_async(
+                        body_file.write,
+                        thread_sensitive=False,
+                    )
+                    # Use async_write
+                    await async_write(message["body"])
+                else:
+                    # Still in memory: safe to write synchronously.
+                    body_file.write(message["body"])
```

Comment:
Given the block comment and the code, I don't think we need the additional line comments here.